src/components/tableDrawing/tablePropertiesDrawingFunctions.js

```diff
@@ -0,0 +1,30 @@
+import { PROPERTIES } from "../../constants/headerContantes/headerConstantes";
+import {
+  changeSelectedValueInView,
+  changeShowCreateTabPopover,
+  changeShowView,
+  changeToggleAddPropertyPopover,
+} from "../../features/showPopoversInfo/showPopoverInfoSlice";
+import {
+  changeToggleAddNewPropertyType,
+  changetoggleEditTypeDrawer,
+  changeToggleSaveNewPropertyField,
+} from "../../features/tableDataInfo/tableDataInfoSlice";
+
+const addProperty = (dispatch) => {
+  dispatch(changeShowCreateTabPopover(true));
+  dispatch(changeSelectedValueInView(PROPERTIES));
+  dispatch(changeShowView(true));
+  dispatch(changeToggleAddPropertyPopover(true));
+  dispatch(changetoggleEditTypeDrawer(true));
+  dispatch(changeToggleAddNewPropertyType(true));
+  dispatch(changeToggleSaveNewPropertyField(true));
+};
+
+export default addProperty;
```

Comment:
why is this the default export?

Reply:
because eslint does not allow only export